third_party/blink/renderer/modules/cobalt/h5vcc_system/h_5_vcc_system.cc

```diff
@@ -63,6 +65,53 @@ void H5vccSystem::OnGetLimitAdTracking(ScriptPromiseResolver* resolver,
   resolver->Resolve(result);
 }
 
+ScriptPromise H5vccSystem::concealOrStop(
+    ScriptState* script_state,
+    const H5vccSystemConcealOrStopCallbacks* callbacks,
+    ExceptionState& exception_state) {
+  auto* resolver = MakeGarbageCollected<ScriptPromiseResolver>(
+      script_state, exception_state.GetContext());
+  EnsureReceiverIsBound();
+
+  remote_h5vcc_system_->GetUserOnExitStrategy(
+      WTF::BindOnce(&H5vccSystem::OnGetUserOnExitStrategy, WrapPersistent(this),
+                    WrapPersistent(resolver), WrapPersistent(callbacks)));
+  return resolver->Promise();
+}
+
+ScriptPromise H5vccSystem::concealOrStop(ScriptState* script_state,
+                                         ExceptionState& exception_state) {
+  return concealOrStop(script_state, /*callbacks=*/nullptr, exception_state);
+}
+
+void H5vccSystem::OnGetUserOnExitStrategy(
+    ScriptPromiseResolver* resolver,
+    const H5vccSystemConcealOrStopCallbacks* callbacks,
+    h5vcc_system::mojom::blink::UserOnExitStrategy result) {
+  if (result == h5vcc_system::mojom::blink::UserOnExitStrategy::kNoExit) {
+    resolver->Resolve(/*concealedOrStopped=*/false);
+    return;
+  }
+
+  auto callback = WTF::BindOnce(&ScriptPromiseResolver::Resolve<bool>,
+                                WrapPersistent(resolver),
+                                /*concealedOrStopped=*/true);
+
```

Comment:
QQ: What happens if the exit strategy is one but there is callback for the other? E.g. UserOnExitStrategy `result` is `kConceal` but there's a `beforeStop()` callback? I guess the other is just ignored?